proposed/standard-ci-env-variables.md

```diff
@@ -23,7 +23,7 @@ It is important that these environment variables do not conflict with other vari
 
 ## Support from CI Services
 
-This plan will only work if CI services decide to support these environment variables. An important question is whether CI services have similar environment variables already. The table below suggests that the information we need is already available.
```

Comment:
Here are the AWS CodeBuild environment variables for an extra data point. We don't have one for STANDARDCI_REPOSITORYNAME.

STANDARDCI_REPOSITORYCOMMITID = CODEBUILD_RESOLVED_SOURCE_VERSION
STANDARDCI_REPOSITORYROOT = CODEBUILD_SRC_DIR
STANDARDCI_REPOSITORYNAME = <not provided>
STANDARDCI_REPOSITORYURI = CODEBUILD_SOURCE_REPO_URL

Reply:
Thanks. Would you be amenable to exposing these new names and also filling the gap on the missing environment variable?

You can add a new column to the table if you'd like, via PR.

Reply:
I can discuss the option to support them with the AWS CodeBuild team but I would need to know more what they would be used for to convince them.

Reply:
Makes sense. We're on the cusp of publishing the design that produced this proposal. That should help.

Reply:
Looking at these names again I'd suggest we insert some more ```_``` to make them more readable:

```
STANDARDCI_REPOSITORY_COMMIT_ID
STANDARDCI_REPOSITORY_ROOT 
STANDARDCI_REPOSITORY_NAME
STANDARDCI_REPOSITORY_URI
```